src/install/ibm-mq/intro.mdx

```diff
@@ -0,0 +1,34 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureIbmmqDashboard from 'images/infrastructure_screenshot-crop_ibmmq-dashboard.webp'
+
+<Callout title="Public preview">
+  This feature is currently a public preview.
+</Callout>
+
+Our IBM MQ [integration](/docs/integrations/host-integrations/getting-started/introduction-host-integrations)
+collects and sends dimensional metrics from [IBM MQ](https://www.ibm.com/products/mq).
+
+You can view this [metric data](#metrics) in pre-built <InlinePopover type="dashboards" />, create <InlinePopover type="alerts" /> policies, and create custom queries and charts.
```

Comment:
I'd remove these popovers. They're nice in a way to introduce users to these NR concepts, but we want as few links out of the doc as possible (so users don't get lost)